epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,268 @@ details:
 }
 ```
 
+#### Total balance update events
+
+After the channel has been opened it has a total balance of tokens committed to
+it. This balance is persisted as part of the on-chain channel state. Upon
+closing a channel on-chain, the closing balances of the participants are
+checked against this balance. Under no circumstances can the sum of the closing balances
+be greater than the total balance on-chain.
+
+Participants are able to modify the total balance: the following two functionalities are available:
+
+* deposit - when a participant wants to commit more tokens from his on-chain
+  balance to the channel total balance
+* withdrawal - when a participant wants to transfer tokens out of the channel
+  on-chain balace to one's personal account
```

Comment:
balace -> balance
Should we mention that a deposit/withdrawal of 0 could work as a mutually agreed snapshot operation?

Reply:
I plan on adding this when describing the solo-snapshot tx